server/src/main/java/org/apache/druid/server/coordinator/duty/KillUnusedSegments.java

```diff
@@ -87,29 +90,36 @@ public class KillUnusedSegments implements CoordinatorDuty
   private final OverlordClient overlordClient;
 
   public KillUnusedSegments(
-      SegmentsMetadataManager segmentsMetadataManager,
-      OverlordClient overlordClient,
-      DruidCoordinatorConfig config
+      final SegmentsMetadataManager segmentsMetadataManager,
+      final OverlordClient overlordClient,
+      final DruidCoordinatorConfig config
   )
   {
     if (config.getCoordinatorKillPeriod().getMillis() < config.getCoordinatorIndexingPeriod().getMillis()) {
-      throw InvalidInput.exception(
-          "druid.coordinator.kill.period[%s] must be >= druid.coordinator.period.indexingPeriod[%s]",
-          config.getCoordinatorKillPeriod(),
-          config.getCoordinatorIndexingPeriod()
-      );
+      throw DruidException.forPersona(DruidException.Persona.OPERATOR)
+                          .ofCategory(DruidException.Category.INVALID_INPUT)
+                          .build(StringUtils.format(
```

Comment:
Nit: would look neater like this:
```suggestion
                          .build(
                              StringUtils.format(
```